streams/test-utils/src/test/java/org/apache/kafka/streams/TopologyTestDriverTest.java

```diff
@@ -1088,4 +1090,68 @@ public void shouldCreateStateDirectoryForStatefulTopology() {
         final TaskId taskId = new TaskId(0, 0);
         assertTrue(new File(appDir, taskId.toString()).exists());
     }
+
+    @Test
+    public void shouldReturnIteratorForMultipleRecords() {
+        testDriver = new TopologyTestDriver(setupSourceSinkTopology(), config);
+
+        testDriver.pipeInput(consumerRecord1);
```

Comment:
nit: I would also create a local `consumerRecord1` as you did with `consumerRecord2`. Alternatively, you could use the global `consumerRecord2` instead of creating a local one. In my opinion, mixing the global and local makes the code less readable.

Reply:
Done
I took the liberty and renamed the methods to "iterable" (from "iterate") because I missed that on the name-change.